StitchCore/StitchCore/Source/Auth/UserProfile.swift

```diff
@@ -33,4 +42,5 @@ public struct UserProfile: Codable {
     public let identities: [Identity]
     /// The extra data associated with this user.
     public let data: [String: String]
+    internal let roles: [Role]?
```

Comment:
I don't know how I feel about making `UserProfile` be the model for both regular Stitch users and Admin users, esp. since this property will be irrelevant for 99% of our users (unless we're adding roles to regular Stitch users too). Should we reject DRY principles here like we did for `LinkInfo` vs. `AuthInfo` and make an `AdminUserProfile` class in `StitchCoreTests`?

Reply:
It is internal though so no external user can use it